src/interfaces.js

```diff
@@ -132,5 +132,5 @@ interface ConfigureRouter {
   /**
   * Implement this hook if you want to add sub-routes to your view.
   */
-  configureRouter(config: RouterConfiguration, router: Router);
+  configureRouter(config: RouterConfiguration, router: Router): any;
```

Comment:
> It's as good as a doc comment.

Yeah, except in code, where it belongs to ;) Also, as You see, You haven't specified it in the doc either.